core/templates/services/site-analytics.service.spec.ts

```diff
@@ -28,558 +28,564 @@ describe('Site Analytics Service', () => {
   beforeEach(() => {
     sas = TestBed.get(SiteAnalyticsService);
     ws = TestBed.get(WindowRef);
-    spyOnProperty(sas, 'CAN_SEND_ANALYTICS_EVENTS', 'get')
-      .and.returnValue(true);
-
-    ws.nativeWindow.gtag = function() {};
-    gtagSpy = spyOn(ws.nativeWindow, 'gtag').and.stub();
   });
 
-  it('should register start login event', () => {
-    const element = 'LoginEventButton';
-    sas.registerStartLoginEvent(element);
+  it('should intialize google analytics', () => {
+    expect(ws.nativeWindow.gtag).toBeDefined();
+  });
 
-    expect(gtagSpy).toHaveBeenCalledWith('event', 'click', {
-      event_category: 'LoginButton',
-      event_label: '/context.html LoginEventButton'
+  describe('Events', () => {
```

Comment:
The thing after 'describe' and the thing after 'it' should join together to form a coherent clause, but "Events should register start login event" does not make sense.

Reply:
changed.
Now the sentence will be "Site Analytics Service when tested using gtag spy should start login event".